ingestion/src/metadata/ingestion/stage/table_usage.py

```diff
@@ -91,6 +94,10 @@ def stage_record(self, record: QueryParserData) -> None:
             return None
         for table in record.tables:
             try:
+                try:
+                    self.table_queries[table].append(SqlQuery(query=record.sql))
+                except KeyError:
+                    self.table_queries[table] = [SqlQuery(query=record.sql)]
```

Comment:
This is a bit dirty hehe

Let's refactor this logic into a separated function "add_table_query" or something like that. Then, check if `table` is in the table_queries keys(). Then append, otherwise initialise the dict